app/src/main/java/com/keystone/cold/ui/fragment/main/AssetFragment.java

```diff
@@ -119,11 +119,12 @@ private void initScanResult() {
                     @Override
                     public void handleScanResult(ScanResult result) throws Exception {
                         if (result.getType().equals(ScanResultTypes.PLAIN_TEXT)) {
-                            if (handleSignElectrumPSBT(result)) return;
+                            if (handleSignPlainTextPSBT(result)) return;
```

Comment:
Only electrum uses Base43 encoding psbt now. It would be better not change the method name.